cmd/client/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+func main() {
+	endpoint := "http://localhost:8080/"
+	// контейнер данных для запроса
+	data := url.Values{}
+	// приглашение в консоли
+	fmt.Println("Введите длинный URL")
+	// открываем потоковое чтение из консоли
+	reader := bufio.NewReader(os.Stdin)
+	// читаем строку из консоли
+	long, err := reader.ReadString('\n')
+	if err != nil {
+		panic(err)
```

Comment:
```suggestion
		log.Fatal(err)
```
Паника немного не для этого сделана.